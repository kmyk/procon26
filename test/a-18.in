11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111011111111111111111
11111111111111011111111111111111
11111111111110001111111111111111
11111111111111011111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111

2
00000000
00000000
00000000
00000000
00000100
00001110
00000100
00000000

00000001
00000000
00000000
00000000
00000000
00000000
00000000
00000000
