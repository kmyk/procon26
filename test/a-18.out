9 10 H 0
14 13 H 270
